test/sequential/rolldown.updateL1FromL1.test.ts

```diff
@@ -534,17 +512,21 @@ describe("updateL1FromL1 - cancelResolution and deposit errors", () => {
   });
 
   beforeEach(async () => {
-    //TODO: Replace this by some monitoring of the active queue.
-    await waitForNBlocks((await Rolldown.disputePeriodLength()).toNumber());
     await SequencerStaking.removeAddedSequencers(10);
     chain = "Ethereum";
-    sequencer = await setupASequencer(chain);
+    [sequencer] = await setupUsers();
+    await SequencerStaking.setupASequencer(sequencer, chain);
     txIndex = await Rolldown.lastProcessedRequestOnL2(chain);
+    waitingPeriod = SequencerStaking.blocksForSequencerUpdate() * 5;
```

Comment:
what is that magic number * 5 ?

Reply:
We needed to replace the number 50 with a number tied to some constant, I chose blocksForSequencerUpdate